sqlengine/mysql_engine.go

```diff
@@ -34,6 +34,11 @@ func (d *MySQLEngine) Open(address string, port int64, dbname string, username s
 	return nil
 }
 
+func (d *MySQLEngine) Ping() error {
+	panic("Not implemented")
```

Comment:
Alternatively, if we don't want to implement this for MySQL at this point, it could simply return `errors.New("Not Implemented")`.

The problem with using `panic` here is that it will cause the broker to exit. This means that if a mysql instance gets deployed for any reason, the broker will cease to function at all until this code is updated.